test/index.js

```diff
@@ -1,7 +1,6 @@
 var assert = require('assert');
 var sinon = require('sinon');
 var mongoose = require('mongoose');
-require('sinon-as-promised');
```

Comment:
It's obsolete as it states on sinon-as-promised page "Sinon 2 added resolves and rejects methods and no longer requires this library."